tpm2/keydata_v0.go

```diff
@@ -188,6 +192,10 @@ func (d *keyData_v0) ValidateData(tpm *tpm2.TPMContext, session tpm2.SessionCont
 	return pcrPolicyCounter, nil
 }
 
+func (d *keyData_v0) DeriveAuthValue(_ []byte) ([]byte, error) {
+	return nil, errors.New("not supported")
+}
```

Comment:
what is this about? this method is not part of keyData, nor I see calls to methods of this name?

Reply:
That's just some code I forgot to delete.